kotlin/token-authentication/README.md

```diff
@@ -1,36 +1,40 @@
 # Token authentication
 
-This sample demonstrates how to access a web map that is secured with ArcGIS token-based authentication.
+Access a web map that is secured with ArcGIS token-based authentication.
 
-![Token authentication](token-authentication.png)
+![Image of token authentication](token-authentication.png)
 
-## How to use the sample
+## Use case
+
+Allows you to access a secure service with the convenience and security of ArcGIS token-based authentication. For example, rather than providing a user name and password every time you want to access a secure service, you only provide those credentials initially to obtain a token which can then be used to access secured resources.
```

Comment:
@hud10837  how about:
```suggestion
You can access a secure service with the convenience and security of ArcGIS token-based authentication. For example, rather than providing a user name and password every time you want to access a secure service, you only provide those credentials initially to obtain a token which can then be used to access secured resources.
```